### PR TITLE
Add atomic scripts to REVSDL-1827. Part 1

### PR DESCRIPTION
**Pull request by VVeremjova**
Splitted failed tests for test set 2, 3 and 8

### CODE REVIEW

**Comment by VVeremjova** [comment]
@Aleksandriys Please review

File: test_scripts/RC/REVSDL-1827/02.01.01-REVSDL-1827_ButtonPress_LeftLongRadio.lua [hunk 1]
```diff
@@ -0,0 +1,123 @@
...
+						self.hmiConnection:SendResponse(data.id, data.method, "SUCCESS", {})
+					end)
+
+					EXPECT_RESPONSE(cid, { success = true, resultCode = "SUCCESS" })
+				end
+			--End Test case CommonRequestCheck.2.1.1
+		-----------------------------------------------------------------------------------------
+--=================================================END TEST CASES 2==========================================================--
+
+return Test
```
Comment (Aleksandriys) [inline test_scripts/RC/REVSDL-1827/02.01.01-REVSDL-1827_ButtonPress_LeftLongRadio.lua]: Please remove return Test.

File: test_scripts/RC/REVSDL-1827/02.01.03-REVSDL-1827_ButtonPress_BackRightLongRadio.lua [hunk 2]
```diff
@@ -0,0 +1,122 @@
...
+					end)
+
+					EXPECT_RESPONSE(cid, { success = true, resultCode = "SUCCESS" })
+				end
+			--End Test case CommonRequestCheck.2.1.3
+
+		-----------------------------------------------------------------------------------------
+--=================================================END TEST CASES 2==========================================================--
+
+return Test
```
Comment (Aleksandriys) [inline test_scripts/RC/REVSDL-1827/02.01.03-REVSDL-1827_ButtonPress_BackRightLongRadio.lua]: Please remove return Test.

Comment (VVeremjova) [inline test_scripts/RC/REVSDL-1827/02.01.03-REVSDL-1827_ButtonPress_BackRightLongRadio.lua]: @Aleksandriys See  c543aef

File: test_scripts/RC/REVSDL-1827/02.01.01-REVSDL-1827_ButtonPress_LeftLongRadio.lua (hunk 1 above)
Comment (VVeremjova) [inline test_scripts/RC/REVSDL-1827/02.01.01-REVSDL-1827_ButtonPress_LeftLongRadio.lua]: @Aleksandriys See  c543aef

**Pull request merged**